### PR TITLE
workspace section added

### PR DESCRIPTION
**Pull request by MarkWoulfeNeo4j**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-aura-264.surge.sh

**Comment by MarkWoulfeNeo4j** [comment]
[Add admonition mentioning workspace](https://trello.com/c/heCu883M/226-add-admonition-mentioning-workspace)

File: modules/ROOT/pages/auradb/getting-started/connect-database.adoc [hunk 1]
```diff
@@ -70,6 +66,26 @@ For more information, see link:{neo4j-docs-base-uri}/bloom-user-guide/current/bl
...
+
+To open an instance with Workspace:
+
+. Navigate to the https://console.neo4j.io/?product=aura-db[Neo4j Aura Console] in your browser.
+. Select the *Open* button on the instance you want to open.
+. Enter the *Database user* and *Password* credentials in the window that opens. 
+These are the same credentials you stored when xref:auradb/getting-started/create-database.adoc[creating the instance].
+. Select *Connect*.
+
+For more information on using Neo4j Workspace, please see the https://neo4j.com/product/workspace/[Product page].
```
Comment (nvitucci) [inline modules/ROOT/pages/auradb/getting-started/connect-database.adoc:81]:
```suggestion
See the https://neo4j.com/product/workspace/[product page] for more information on using Neo4j Workspace.
```

File: modules/ROOT/pages/auradb/getting-started/connect-database.adoc [hunk 2]
```diff
@@ -134,4 +150,4 @@ For help on a specific command type:
     :help command
 ----
 
-For more information on Cypher Shell, including how to install it, please see the link:{neo4j-docs-base-uri}/operations-manual/current/tools/cypher-shell/[Cypher Shell documentation].
+For more information on Cypher Shell, including how to install it, please see the link:{neo4j-docs-base-uri}/operations-manual/current/tools/cypher-shell/[Cypher Shell documentation].
```
Comment (nvitucci) [inline modules/ROOT/pages/auradb/getting-started/connect-database.adoc:153]:
```suggestion
For more information on Cypher Shell, including how to install it, see the link:{neo4j-docs-base-uri}/operations-manual/current/tools/cypher-shell/[Cypher Shell documentation].
```

File: modules/ROOT/pages/auradb/getting-started/connect-database.adoc (hunk 1 above)
Comment (MarkWoulfeNeo4j) [inline modules/ROOT/pages/auradb/getting-started/connect-database.adoc:81]:
```suggestion
For more information on using Neo4j Workspace, see the https://neo4j.com/product/workspace/[Product page].
```

**Review by nvitucci — APPROVED** [review]

**Pull request merged**